packages/ui-tag/src/Tag/styles.ts

```diff
@@ -59,7 +59,7 @@ const generateStyle = (componentTheme: TagTheme, props: TagProps): TagStyle => {
       text: {
         lineHeight: `calc(${componentTheme.heightMedium} - (${componentTheme.defaultBorderWidth} * 2))`
       },
-      icon: { fontSize: '0.75rem' }
```

Comment:
The "small" version is still a few px lower :) Also, I guess this PR needs a +1 from Ákos as well :D

Reply:
Akos has given his +1, he told me that we only use the "normal" size